test/corpus/packages.txt

```diff
@@ -2,17 +2,13 @@
 Packages
 ==================
 
-package wasmcloud:hello@1.0.0;
-package wasi:io;
```

Comment:
This syntax is [still a part of the spec](https://github.com/WebAssembly/component-model/blob/main/design/mvp/WIT.md#package-names). Please do not remove it from the test cases.

Reply:
True but you can only have a single package declaration in a source file. Added more, separate tests for package declarations.